commit e0dcc22981258735b767e446cf3e8f4ecf1fba06
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-04-23T08:10:35-07:00 (6 months ago)
    
    TRITON-1622 Instance migration tests should make use of multiple servers
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/lib/backends/smartos/tasks/machine_migrate.js b/lib/backends/smartos/tasks/machine_migrate.js
index 9eb0725..1ed40af 100644
--- a/lib/backends/smartos/tasks/machine_migrate.js
+++ b/lib/backends/smartos/tasks/machine_migrate.js
@@ -413,16 +413,6 @@ function setupFilesystem(callback) {
     // Check for override of the vm uuid. Note that performing an uuid override
     // is only supported in a non-production environment.
     if (vm_uuid !== target_vm_uuid) {
-        var fileMarkerPath = '/lib/sdc/.sdc-test-no-production-data';
-        try {
-            fs.accessSync(fileMarkerPath);
-        } catch (ex) {
-            this.fatal('Cannot perform migration uuid override - ' +
-                'as non-production marker file is missing: ' +
-                fileMarkerPath);
-            return;
-        }
-
         dataset = dataset.replace(vm_uuid, target_vm_uuid);
 
         log.warn({
@@ -490,16 +480,6 @@ function removeSyncSnapshots(callback) {
     // Check for override of the vm uuid. Note that performing an uuid override
     // is only supported in a non-production environment.
     if (vm.uuid !== payload.vm_uuid) {
-        var fileMarkerPath = '/lib/sdc/.sdc-test-no-production-data';
-        try {
-            fs.accessSync(fileMarkerPath);
-        } catch (ex) {
-            self.fatal('Cannot perform migration uuid override - ' +
-                'as non-production marker file is missing: ' +
-                fileMarkerPath);
-            return;
-        }
-
         datasets = datasets.map(function (aDataset) {
             return aDataset.replace(vm.uuid, payload.vm_uuid);
         });
diff --git a/package.json b/package.json
index 9fed610..bffe8dd 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cn-agent",
   "description": "Triton Compute Node Agent",
-  "version": "2.12.2",
+  "version": "2.12.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
